Example/Tests/BPKFontTest.m

```diff
@@ -102,26 +102,36 @@ - (void)testAttributedStringWithCustomFontFaces {
 }
 
 - (void)testFontWithStyle {
-    /*
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextXs], [BPKFont textXs]);
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextXsEmphasized], [BPKFont textXsEmphasized]);
-
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextSm], [BPKFont textSm]);
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextSmEmphasized], [BPKFont textSmEmphasized]);
-
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextBase], [BPKFont textBase]);
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextBaseEmphasized], [BPKFont textBaseEmphasized]);
-
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextLg], [BPKFont textLg]);
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextLgEmphasized], [BPKFont textLgEmphasized]);
-
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextXl], [BPKFont textXl]);
-    XCTAssertEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextXlEmphasized], [BPKFont textXlEmphasized]);
-
-    // Small sanity checks
-    XCTAssertNotEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextXl], [BPKFont fontWithStyle:BPKFontStyleTextLg]);
-    XCTAssertNotEqualObjects([BPKFont fontWithStyle:BPKFontStyleTextSm], [BPKFont fontWithStyle:BPKFontStyleTextBase]);
-     */
+    BPKFontStyle styles[] = {
+        BPKFontStyleTextBase,
+        BPKFontStyleTextBaseEmphasized,
+        BPKFontStyleTextCaps,
+        BPKFontStyleTextCapsEmphasized,
+        BPKFontStyleTextLg,
+        BPKFontStyleTextLgEmphasized,
+        BPKFontStyleTextSm,
+        BPKFontStyleTextSmEmphasized,
+        BPKFontStyleTextXl,
+        BPKFontStyleTextXlEmphasized,
+        BPKFontStyleTextXlHeavy,
+        BPKFontStyleTextXs,
+        BPKFontStyleTextXsEmphasized,
+        BPKFontStyleTextXxl,
+        BPKFontStyleTextXxlEmphasized,
+        BPKFontStyleTextXxlHeavy,
+        BPKFontStyleTextXxxl,
+        BPKFontStyleTextXxxlEmphasized,
+        BPKFontStyleTextXxxlHeavy
+    };
+
+    for (NSUInteger i = 0; i < sizeof(styles) / sizeof(styles[0]); i++) {
```

Comment:
Could we just use `for (BPKFontStyle style in styles) { ... }`?

Reply:
Not with C arrays no

Reply:
Why is it a C array tho?

Reply:
Because otherwise we'd have to box it in `NSNumber` to put them in an `NSArray`